package.json

```diff
@@ -32,7 +32,8 @@
     "dayjs": "^1.11.13",
     "dotenv": "^16.4.7",
     "tsx": "^4.19.2",
-    "yahoo-finance2": "^2.13.3"
+    "yahoo-finance2": "^2.13.3",
+    "ts-md5": "^1.3.1"
```

Comment:
You can also hash with plain node, not requiring an additional library. Like below:

```ts
import * as crypto from “crypto”;
var name = 'braitsch';
var hash crypto.createHash('md5').update(name).digest('hex');
```

Reply:
ah, ok. I'm not an experienced typescript coder. It's my first ts ever. So, I went for an approach which googled.